pom.xml

```diff
@@ -0,0 +1,283 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>uk.gov.companieshouse</groupId>
+        <artifactId>companies-house-parent</artifactId>
+        <version>1.3.1</version>
+    </parent>
+
+    <name>officers-search-consumer</name>
+    <description>This service will consume from the stream-company-officers kafka topic. Messages consumed will be processed by sending requests to search.api.ch.gov.uk to update the primary index.</description>
+
+    <artifactId>officers-search-consumer</artifactId>
+    <version>latest</version>
+    <packaging>jar</packaging>
+
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <spring.boot.version>2.7.10</spring.boot.version>
```

Comment:
The latest Spring Boot version is now `2.7.11`. You can check the latest from this page https://start.spring.io/